ducktape/cluster/remoteaccount.py

```diff
@@ -62,6 +63,8 @@ def _can_ping_url(self, url, headers):
     def ssh_command(self, cmd):
         if self.local:
             return cmd
+        if self.sudo_user is not None:
```

Comment:
@lindong28 Thanks, for your clarifications. One followup question is - what would this enable you to do that you cannot do today? Or, what specific thing is hard for you to do now, that this would make easier?

As for what this looks like api-wise, here's some thinking out loud about what this should maybe look like:

specify sudo per ssh command might look something like:
*  `node.account.ssh(cmd, super_user="userB")` or maybe
*  `node.account.ssh(cmd, sudo=True)` along with similar flag on other remote account methods
  * In this case, `RemoteAccount` instance as `super_user` field which by default is `root`

My take is the second version is better; the troublesome thing about specifying `super_user` in test code, is that the value of `super_user` is probably dependent on the environment in which the tests are run, so I agree with you that it would make sense to tie super_user to the `RemoteAccount` instance.

What do you think?